### DISCUSSION TITLE
Svg object using Android

### DISCUSSION THREAD

**Discussion by JTrotta** [Mapsui/Mapsui#2424] [Q&A]
Cannot figure out how to use svg in Android.
Any example or help?
Thanks

**Comment by charlenni** [comment]
Some more information would be helpful. You want to display a SVG on the Map? Or you want to display it as a marker? Or as a button widget? Many possibilities.

**Reply by JTrotta** [reply]
You're right. 
A moving marker , using lat e lon from gps
